config.yml.sample

```diff
@@ -74,5 +74,5 @@ slaves:
 # List of builders whose failures should not be reported.  Tests listed here
 # will be reported in a separate section of the Buildbot web page, to indicate
 # that they are expected failures.
-failing_builders: []
+failing_builders: ['flocker/functional/storage-driver']
```

Comment:
This can probably be left out of the final branch.  This isn't actually our deployment configuration, just a template to give folks an idea of what a valid configuration looks like.  We will want to mark new, failing builders as expected to fail in our real configuration (which is maintained in lastpass).